ostree_upload_server/task/receive.py

```diff
@@ -3,10 +3,15 @@
 
 from gevent.subprocess import check_output, CalledProcessError, STDOUT
 
+from ostree_upload_server.flatpak_importer import FlatpakImporter
 from ostree_upload_server.repolock import RepoLock
 from ostree_upload_server.task.base import BaseTask
 from ostree_upload_server.task.state import TaskState
 
+import gi
+gi.require_version('OSTree', '1.0')
```

Comment:
This isn't needed since you're not importing `OSTree`.

Reply:
Good point - moved the code that uses it out.